images/router/haproxy/conf/haproxy-config.template

```diff
@@ -193,7 +193,7 @@ frontend public
     {{- end }}
   {{- if isTrue (env "ROUTER_USE_PROXY_PROTOCOL") }} accept-proxy{{ end }}
   mode http
-  tcp-request inspect-delay 5s
+  tcp-request inspect-delay {{env "ROUTER_INSPECT_DELAY" "5s"}}
```

Comment:
nit: it seems the vast majority of env substitutions have spaces. e.g. `{{ env "ROUTER..." <val> }}` - can we go with the majority here and on the other additions?

Reply:
will do